src/WebView.js

```diff
@@ -175,6 +176,24 @@ class WebView extends WebKit.WebView {
     }
     return false;
   };
+
+  #onContextMenu = (webView, contextMenu, event, hitTestResult) => {
+    const itemsToRemove = [
+      WebKit.ContextMenuAction.STOP,
+      WebKit.ContextMenuAction.RELOAD,
+      WebKit.ContextMenuAction.INSPECT_ELEMENT,
```

Comment:
I think we had intentionally turned on ``enable-developer-extras`` which you can find in WebView.blp. I don't remember why we enabled it but we can disable it there for now